internal/cli/clients/client.go

```diff
@@ -1737,7 +1778,7 @@ func getCredentials(log logr.Logger, ctx context.Context) (string, string, error
 	return user, pass, nil
 }
 
-func getCerts(ctx context.Context, log logr.Logger) (string, error) {
+func getCerts(log logr.Logger, ctx context.Context) (string, error) {
```

Comment:
only mentioning this, since it's a private func no need to change it:

> Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx: 

https://pkg.go.dev/context#pkg-overview

Reply:
Heh. And here my change brought this one function into sync with the others in the file.

Reply:
Fixed with commit c7c989546ecc858c5df015650247b70c76d1be0c